ci-operator/step-registry/aws/provision/edge-zone-subnet/aws-provision-edge-zone-subnet-commands.sh

```diff
@@ -378,6 +396,53 @@ elif [[ ${EDGE_ZONE_TYPES} == "wavelength-zone" ]]; then
   else
     echo $wavelengthzone_priv_subnet > $SHARED_DIR/edge_zone_subnet_id
   fi
+elif [[ ${EDGE_ZONE_TYPES} == "outpost" ]]; then
+  # Subnet
+  STACK_NAME="${CLUSTER_NAME}-outpost"
+  echo ${STACK_NAME} >> "${SHARED_DIR}/to_be_removed_cf_stack_list"
+  extra_options=" "
+  outpost_subnet_output="$ARTIFACT_DIR/outpost_subnet_output.json"
+  outpost_subnet_params="$ARTIFACT_DIR/outpost_subnet_params.json"
+
+  outpost_arn=$(jq -r '.OutpostArn' ${CLUSTER_PROFILE_DIR}/aws_outpost_info.json)
+  outpost_az=$(jq -r '.AvailabilityZone' ${CLUSTER_PROFILE_DIR}/aws_outpost_info.json)
+  echo $outpost_az > ${SHARED_DIR}/outpost_availability_zone
+
+  private_route_table_id=$(head -n 1 "${SHARED_DIR}/private_route_table_id") # TBD
+  public_route_table_id=$(head -n 1 "${SHARED_DIR}/public_route_table_id")
+
+  priv_subnet_cidr="10.0.129.0/24"
+  pub_subnet_cidr="10.0.128.0/24"
```

Comment:
Does the cidr is bound to the specific outpost ?

Reply:
No, it's part of VPC CIDR, same as WLZ and LZ